From 5d42e09997396c41e42fdc9e53fda5adf68ed523 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Sat, 29 Sep 2012 10:08:02 +0800
Subject: [PATCH 069/281] ENGR00226374 init.rc: import usb.rc by ro.hardware
 property

import usb.rc by name of the ro.hardware property
init.usb.${ro.hardware}.rc not by hardcode.

Change-Id: Id76027711566f010012a4655f1c7504abe76e1d5
Signed-off-by: Xinyu Chen <xinyu.chen@freecale.com>
---
 arm2_6dq/init.rc     | 3 ++-
 arm2_6sl/init.rc     | 3 ++-
 imx53_smd/init.rc    | 3 ++-
 imx6/sabresd/init.rc | 3 ++-
 sabreauto_6q/init.rc | 3 ++-
 5 files changed, 10 insertions(+), 5 deletions(-)

diff --git a/arm2_6dq/init.rc b/arm2_6dq/init.rc
index f06dd59..d51478c 100755
--- a/arm2_6dq/init.rc
+++ b/arm2_6dq/init.rc
@@ -1,4 +1,5 @@
-import init.freescale.usb.rc
+import init.${ro.hardware}.usb.rc
+
 on boot
 
     # Set permission for IIM node
diff --git a/arm2_6sl/init.rc b/arm2_6sl/init.rc
index 6fc6083..f6b19eb 100755
--- a/arm2_6sl/init.rc
+++ b/arm2_6sl/init.rc
@@ -1,4 +1,5 @@
-import init.freescale.usb.rc
+import init.${ro.hardware}.usb.rc
+
 on boot
 
     # Set permission for IIM node
diff --git a/imx53_smd/init.rc b/imx53_smd/init.rc
index 3351d16..908f4ac 100755
--- a/imx53_smd/init.rc
+++ b/imx53_smd/init.rc
@@ -1,4 +1,5 @@
-import init.freescale.usb.rc
+import init.${ro.hardware}.usb.rc
+
 on boot
 
     # Set permission for IIM node
diff --git a/imx6/sabresd/init.rc b/imx6/sabresd/init.rc
index 6174a42..bb275e4 100755
--- a/imx6/sabresd/init.rc
+++ b/imx6/sabresd/init.rc
@@ -1,4 +1,5 @@
-import init.freescale.usb.rc
+import init.${ro.hardware}.usb.rc
+
 on boot
 
     # Set permission for IIM node
diff --git a/sabreauto_6q/init.rc b/sabreauto_6q/init.rc
index 57eb2d1..4d7043c 100644
--- a/sabreauto_6q/init.rc
+++ b/sabreauto_6q/init.rc
@@ -1,4 +1,5 @@
-import init.freescale.usb.rc
+import init.${ro.hardware}.usb.rc
+
 on boot
 
     # Set permission for IIM node
-- 
1.8.0

